test/distributions/dm/s7.compopts

```diff
@@ -0,0 +1 @@
+--no-print-callstack-on-error
```

Comment:
why?

Reply:
Because the test sorts its output and so the call stack is pretty obscured anyway. I didn't want it to be a maintenance burden.

Reply:
Right, you mentioned.